apps/explorer/src/ui/LinkGroup.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Link } from './Link';
+
+export type LinkGroupProps = {
+    title: string;
+} & ({ text: string | null } | { links: { text: string; to: string }[] });
+
+export function LinkGroup(props: LinkGroupProps) {
```

Comment:
Is this actually something we have in multiple parts of the app? Glad it's generalized but feels like it doesn't really belong in `~/ui` (unless this design exists in other places I just haven't seen yet).

Reply:
I haven't seen it anywhere else will move it to the module functions folder and if we need it we can move it here later